docs/knowledge_center/faq.rst

```diff
@@ -3,43 +3,131 @@
+        - One step further is to adjust existing or create new components (:ref:`Custom component tutorial <components_custom_components_label>` and :ref:`Documentation on how to contribute <developing_label>`)
+
+    .. dropdown:: How can I plot log(p)-h and T-s diagrams from my simulation?
+
+        Fluid property diagrams can help you analyze and improve your
+        thermodynamic processes. CoolProp offers a somewhat unwieldly
+        `inbuilt feature <https://coolprop.org/coolprop/python-plotting.html>`__
+        for creating those, but we recommend using the officially supported
+        companion module fluprodia. It allows for the creation and
+        customization of different types of diagrams for all pure and
+        pseudo-pure fluids available in CoolProp. There is an introductory
+        :ref:`FluProDia Example <fluprodia_label>` that shows its usage as well
+        as TESPy's API to extract the necessary plotting data. A thorough
+        explanation can be found in
+        `fluprodia's online documentation <https://https://fluprodia.readthedocs.io/en/latest/>`__.
+
+    .. dropdown:: How can I plot Q-T diagrams of heat exchangers?
+
+        If you want to analyze the heat transfer within a heat exchangerk, e.g.
+        to investigate the phase change of a working fluid, creating a Q-T
+        diagram can be very helpful. While the heat transfer of simple heat
+        exchanger models like the
+        :py:class:`HeatExchanger <tespy.components.heat_exchangers.base.HeatExchanger>`
+        or :py:class:`Condenser <tespy.components.heat_exchangers.condenser.Condenser>`
+        is easily calculated by hand, this can not be said about more complex
+        variants like the
+        :py:class:`SectionedHeatExchanger <tespy.components.heat_exchangers.sectioned.SectionedHeatExchanger>`
+        or :py:class:`MovingBoundaryHeatExchanger <tespy.components.heat_exchangers.movingboundary.MovingBoundaryHeatExchanger>`.
+        Fortunately, the latter have the
+        :py:meth:`.calc_sections() <tespy.components.heat_exchangers.sectioned.SectionedHeatExchanger.calc_sections>`
+        method that returns useful information about their heat transfer. Below
+        is an example snippet of how to generate a Q-T diagram from such a heat
+        exchanger.
+
+        .. code-block:: python
+
+            from matplotlib import pyplot as plt
+
+            ...
+
+            heatex = SectionedHeatExchanger("heat exchanger")  # or MovingBoundaryHeatExchanger()
+
+            ...
+
+            nw.solve("design")
+
+            heat, T_hot, T_cold, heat_per_section, td_log_per_section = heatex.calc_sections()
+            heat /= 1e6
+
+            fig, ax = plt.subplots(figsize=(10, 6))
+
+            ax.plot(heat, T_hot, "o-", color="red")
+            ax.plot(heat, T_cold, "o-", color="blue")
+
+            ax.set_ylabel("temperature in K")
+            ax.set_xlabel("heat transferred in MW")
+
+            plt.show()
```

Comment:
It would be nice if theres were testable. Maybe we can do the following:

```python
>>> from tespy.tools import HeatExchangerTestNetwork (or from where-ever)
>>> nw = HeatExchangerTestNetwork()
>>> heatex = nw.get_comp("heatex")
>>> nw.solve("design")
>>> ...
>>> [the plotting code]
```
Lines with >>> in front will automatically be tested by the testing pipeline, so we would be notified if anything breaks with this example.

Reply:
True!

But: I kind of like the idea to just grab a TestNetwork from the testnetworks module whenever I need one for fiddling around... Once you get used to them, these could be very useful for quick testing or showing things. You do not need to do all the imports etc..

I'd say: You decide what you prefer :)